mypy/checkexpr.py

```diff
@@ -1485,7 +1485,17 @@ def check_call_expr_with_callee_type(
         )
         proper_callee = get_proper_type(callee_type)
         if isinstance(e.callee, (NameExpr, MemberExpr)):
-            self.chk.warn_deprecated_overload_item(e.callee.node, e, target=callee_type)
+            node = e.callee.node
+            if (
+                (node is None)
+                and (member is not None)
+                and isinstance(object_type, Instance)
+                and ((symbol := object_type.type.names.get(member)) is not None)
```

Comment:
```suggestion
                node is None
                and member is not None
                and isinstance(object_type, Instance)
                and (symbol := object_type.type.names.get(member)) is not None
```

Reply:
I will have a look at it.

Reply:
It became a one-liner followed by a loop.